sci-physics/geant-vmc/geant-vmc-4.3.6.ebuild

```diff
@@ -23,18 +23,19 @@ LICENSE="GPL-2"
 SLOT="4"
 IUSE="doc examples geant3 +g4root +mtroot vgm test"
 
-# sci-physics/root[root7] flag activates std=c++14, which we could only support if also Geant is built with it.
```

Comment:
Since you're fixing this in revbump, there's no need to edit the old version.

Reply:
Would you prefer if I revert this change, or leave as-is (since you already approved the changes)?